lcc_lokavaluto_app_connection/models/res_partner_backend.py

```diff
@@ -1,8 +1,10 @@
 from odoo import models, fields, api
 from odoo.exceptions import UserError
-
+from odoo.tools.safe_eval import safe_eval
 import logging
 
+
+
```

Comment:
unecessary, some pass with ruff in incrementally can be applied at the end if you wish